rfcs/002-store-fees-on-user-and-collective-ledgers.md

```diff
@@ -0,0 +1,72 @@
+# REQUEST FOR COMMENTS 001 - Store fees on the User & Collective ledgers
+
+## Overview
+
+Every operation of moving funds from one ledger to another is
+currently represented as two rows in the `Transactions` table. One to
+add up to the Collective ledger (CREDIT) and the other to subtract
```

Comment:
For completeness, I'd rephrase this as: Each row represents a change in one ledger only. Currently, we only represent two ledgers in the system: Collective and User (also a Collective) and fees awkwardly hang out on both.

Reply:
Ignore this comment, it's covered in next RFC

Reply:
OK Thanks!